sdks/java/core/src/main/java/org/apache/beam/sdk/options/ValueProviders.java

```diff
@@ -1,57 +0,0 @@
-/*
- * Licensed to the Apache Software Foundation (ASF) under one
- * or more contributor license agreements.  See the NOTICE file
- * distributed with this work for additional information
- * regarding copyright ownership.  The ASF licenses this file
- * to you under the Apache License, Version 2.0 (the
- * "License"); you may not use this file except in compliance
- * with the License.  You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-package org.apache.beam.sdk.options;
-
-import static com.google.common.base.Preconditions.checkNotNull;
-
-import com.fasterxml.jackson.databind.node.ObjectNode;
-import java.io.IOException;
-import java.util.Map;
-
-/**
- * Utilities for working with the {@link ValueProvider} interface.
- */
-public class ValueProviders {
```

Comment:
Users may be using ValueProviders since it is a public class in org.apache.beam.sdk.options. Are we concerned about backwards compatability issues?

Reply:
I doubt that anybody was using this, but ok, reverted and deprecated instead.